common/src/main/java/edu/snu/nemo/common/coder/DecoderFactory.java

```diff
@@ -84,6 +85,11 @@ public Decoder create(final InputStream inputStream) {
       return dummyDecoder;
     }
 
+    @Override
+    public Object getCoder() {
```

Comment:
When is this method used? Do we really need this method for all factories?
Because it returns `Object`, users who use `DecoderFactory` can't expect any general functionality from this code.
If it is needed just for key coder factory, it would be enough to implement this in `BeamDecoderFactory` and `SparkDecoderFactory` specifically, because they can be referenced in compiler frontend.